modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/metadata/MetadataManagerImplTest.java

```diff
@@ -51,7 +51,7 @@ public void testGetRequestPath() {
 
 		Assert.assertEquals(
 			"/c/portal/login",
-			_samlHttpRequestUtilImpl.getRequestPath(mockHttpServletRequest));
+			_samlHttpRequestImpl.getRequestPath(mockHttpServletRequest));
```

Comment:
@GabrielHenriquedev It seems this test only tests the extracted class, please rename it.